frontend/components/group/group-droplet-tile.tsx

```
@@ -1,10 +1,19 @@
+"use client";
+
 import { AuthorizedUser, Droplet } from "@/types";
 import { Badge } from "@/components/ui/badge";
-import { Card, CardHeader, CardTitle } from "@/components/ui/card";
+import {
+  Card,
+  CardContent,
+  CardDescription,
```

ESLint

[error] 'CardDescription' is defined but never used.

(@typescript-eslint/no-unused-vars)